examples/runFasterCode.ts

```diff
@@ -1,25 +1,23 @@
-/*  This is an example of how to invoke mariner
+/*  This is an example of how to invoke mariner using the new GraphQL code.
     It mostly exists to be able to test and debug while working on the library code
 
     To use it, you will need to add a variable to your environment:
-        export GITHUB_TOKEN="<token here>"
+        export MARINER_GITHUB_TOKEN="<token here>"
     Optionally, you can have environment variables for INPUT_FILE_PATH and OUTPUT_FILE_PATH,
         but they have defaults that will work with the standard development environment.
     Then, run `npm run build`
-    Finally, run `node dist/indexExample.js`
+    Finally, run `node dist/examples/runFasterCode.js`
 
-    FYI, if it's running correctly, your first logged output in the console
-    will likely say "ERROR". Sorry about that ;-)
-        You'll know it's run correctly if you have a new file
-        exampleData/output.json with some GitHub issues in it.
+    You'll know it's run correctly if you have a new file
+        examples/output.json with some GitHub issues in it.
 */
 
 import fs from 'fs';
-import * as mariner from './mariner/index'; // This is used during development
+import * as mariner from '../src/mariner/index'; // This is used during development
 // import * as mariner from 'oss-mariner'    // This is how the npm package would normally be used
 
 import * as path from 'path';
-import { IssueFinder } from './issueFinder';
+import { IssueFinder } from '../src/issueFinder';
```

Comment:
I'm not sure this is worth fixing now, because the problem was pre-existing. But we should be accessing issueFinder via mariner, which means mariner should be exporting it (if it isn't already). When using the npm package, the same problem as described in the comments above would apply--../src/issueFinder wouldn't exist.